Web-App/src/App.js

```diff
@@ -51,7 +51,7 @@ function App() {
         
         if (config.hasOwnProperty("completionMessage")) {
```

Comment:
Reset completionMessage in `resetSession`
```suggestion
        if (config.hasOwnProperty("completionMessage")) {
          setCompletionMessage(option.completionMessage);
        } else {
          setCompletionMessage("Thanks for your feedback!");
        }
```

Reply:
Maybe we should call `resetSession` when initialising the app?